app/models/draft.rb

```diff
@@ -15,4 +16,9 @@ def display_short_name
   def default_values
     self.draft ||= {}
   end
+
+  def set_native_id
+    self.native_id ||= "mmt_collection_#{id}"
```

Comment:
Moving this makes sense but I don't think we want variables having a native id with `collection` in it. Maybe this method should use the draft_type or class to determine this portion of the string?